apps/style/code-studio/petition.scss

```diff
@@ -0,0 +1,116 @@
+.petition_block {
+  width: 970px;
+  display: flex;
+  flex-flow: column wrap;
+  text-align: center;
+  background-color: white;
+  padding: 10px;
+}
+
+.petition_message {
+  margin: 0;
+  font-size: 21px;
+  line-height: 28px;
+  font-family: 'Gotham 5r', sans-serif;
+  color: #5B6770;
+}
+
+.petition_sign_message {
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: 400;
+  margin: 10px 0 20px 0;
+  font-family: 'Gotham 4r', sans-serif;
+  color: #5B6770;
+}
+
+.petition_form {
+  font-size: 12px;
+  line-height: 18px;
+  color: #dcdcdc;
+  text-align: left;
+  display: flex;
+  flex-flow: row wrap;
+  flex: 1 100%;
+  justify-content: space-around;
+}
+
+input.field, select.field {
+  background-color: #e2e4e3;
+  color: #595959;
+  padding: 5px;
+  border-width: 1px;
+}
+
+input.grey_input {
+  width: 150px;
+}
+
+span.grey_input {
+  width: 160px;
+}
+
+select.dropdown {
+  width: 93px;
+}
+
+span.dropdown {
+  width: 130px;
+}
+
+.dropdown_label {
+  margin: 0 4px 0 0;
+  color: #5B6770;
+  width: 100px;
+  font-size: 14px;
+  vertical-align: middle;
+}
+
+.go_button {
+  min-width: 100px;
+  margin: 0;
+  padding: 4px 12px;
+  height: 30px;
+  font-family: 'Gotham 5r', sans-serif;
+}
+
+.help, a {
```

Comment:
using a comma here applies the rule to both `.help` and `a`, did you mean `.help a` (which would affect any links inside .help elements)?

Reply:
Most of this needs to apply to the help text that are *not* links. But if I don't include the `a` tag specifically, then the color and text decoration get overridden

Reply:
As per discussion below, changing this too to be .help, .help a